app/components/primer/beta/link.html.erb

```diff
@@ -0,0 +1,24 @@
+<% link_content = capture do %>
+  <%= render(Primer::BaseComponent.new(**@system_arguments)) do -%>
+    <% if leading_visual %>
+      <span class="Link-visual Link-leadingVisual">
```

Comment:
Are these css classes defined anywhere?

Reply:
I use them inside the [tests](https://github.com/primer/view_components/pull/2982/files#diff-723673d8c251e05a51a562ffae0fd99e96d2ad3f0b792fa45284f1eb05a6bb56R103). I was unsure what the correct approach is here. Actually, I wanted to use the `data-test-selector` attribute, but I saw that all other tests are refering to the classes.

Reply:
Update: I saw that there was even a test failing because the classes were unused. So I removed them and changed the test accordingly.